src/pocketmine/plugin/PluginDescription.php

```diff
@@ -76,6 +79,8 @@ private function loadMap(array $plugin){
 			throw new PluginException("Invalid PluginDescription main, cannot start within the PocketMine namespace");
 		}
 
+		$this->compatibleMcpeProtocols = array_map(function($v){ return (int) $v; }, $plugin["mcpe-protocol"] ?? $plugin["mcpe-protocols"] ?? []);
```

Comment:
Will PhpStorm try to insert two line feeds when you auto reformat 🙈 
Can actually be replaced with `array_map("intval",`.